string mapID
---
bool response
string errorMessage